### PR TITLE
Add detailed instructions for setting up archive node using Docker and Docker Compose

### PR DESCRIPTION
**Pull request by MartinMinkov**
This commit adds comprehensive instructions for setting up and running the archive node using Docker and Docker Compose. This provides users with alternative methods for running the archive node, which can be more convenient and easier to manage.

### CODE REVIEW

File: docs/node-operators/archive-node.mdx [hunk 1]
```diff
@@ -106,6 +106,145 @@ To run a local archive node to run it in the foreground for testing:
...
+       .....
+     --archive-address 3086\
+   ```
+
+## Set up the archive node using Docker Compose
+
+Docker compose is a tool for defining and running multi-container Docker applications.
+With Compose, you use a YAML file to configure your application's services. Then, with a single command, you create and start all the services from your configuration.
+
+If you wish to run the archive node using Docker Compose, you can use the following instructions.
```
Comment (barriebyron) [inline docs/node-operators/archive-node.mdx:178]:
```suggestion
To run the archive node using Docker Compose, use the following instructions.
```

File: docs/node-operators/archive-node.mdx [hunk 2]
```diff
@@ -106,6 +106,145 @@ To run a local archive node to run it in the foreground for testing:
...
+
+To get started with installing and running the archive node using Docker Compose, follow the steps below.
```
Comment (barriebyron) [inline docs/node-operators/archive-node.mdx:180]:
```suggestion
To run the archive node using Docker Compose:
```

File: docs/node-operators/archive-node.mdx [hunk 3]
```diff
@@ -106,6 +106,145 @@ To run a local archive node to run it in the foreground for testing:
...
+
+   ```
+   mina daemon \
+       .....
+     --archive-address 3086\
+   ```
+
+## Set up the archive node using Docker Compose
+
+Docker compose is a tool for defining and running multi-container Docker applications.
```
Comment (barriebyron) [inline docs/node-operators/archive-node.mdx:175]:
```suggestion
Docker Compose is a tool for defining and running multi-container Docker applications.
```

**Review by barriebyron — APPROVED** [review]
@MartinMinkov  so clear, thank you!! 
I did not test the steps, so this approval is just for editorial content.

**Comment by barriebyron** [comment]
@LucidSamuel  are you up for testing these steps?

**Comment by MartinMinkov** [comment]
>@MartinMinkov so clear, thank you!!
I did not test the steps, so this approval is just for editorial content.

For what it's worth, I tested them on my machine and it's all working. Would love for someone else to try it out though!

**Review by LucidSamuel — APPROVED** [review]
Just tested and works for me too, apologies for the delay.

**Pull request merged**